### ISSUE TITLE
failing test for SeqTools/SeqUtils.t

### ISSUE THREAD

**Issue by hlapp** [bioperl/bioperl-live#296]
I get the following failures against master (on Ubuntu 14.04, using the bioperl/bioperl-deps container as basis).

```
t/SeqTools/SeqUtils.t ............... 1/133 
------------- EXCEPTION: Bio::Root::Exception -------------
MSG: Providing an array of features to Bio::Seq add_SeqFeature() is deprecated and will be removed in a future version. Add a single feature at a time instead.
STACK: Error::throw
STACK: Bio::Root::Root::throw /usr/local/src/bioperl-build/blib/lib/Bio/Root/Root.pm:447
STACK: Bio::Root::RootI::deprecated /usr/local/src/bioperl-build/blib/lib/Bio/Root/RootI.pm:247
STACK: Bio::Seq::add_SeqFeature /usr/local/src/bioperl-build/blib/lib/Bio/Seq.pm:1136
STACK: t/SeqTools/SeqUtils.t:390
-----------------------------------------------------------
# Looks like your test exited with 255 just after 60.
t/SeqTools/SeqUtils.t ............... Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 73/133 subtests 
```

**Comment by hlapp** [comment]
I think this must be a consequence of #294.

**Comment by carandraug** [comment]
Fixed eariler today with a073c593ca018f1a3bd5a834a3f651336387c1d1

> I think this must be a consequence of #294.

I did nothing about #294 yet, I'm only suggesting deprecating that feature.
